src/authMachine.ts

```diff
@@ -29,19 +22,25 @@ import {
 } from './sourceMachine';
 import { storage } from './localCache';
 import { gQuery } from './utils';
-import { GetSourceFileSsrQuery } from './graphql/GetSourceFileSSR.generated';
+import { SourceRegistryData } from './types';
+import { NextRouter } from 'next/router';
 
 const authModel = createModel(
   {
-    client: null! as SupabaseClient,
+    client: createClient(
```

Comment:
Moving it here fixes andarist/sta-580-refactor-auth-machine-to-prevent which made our app spam the Supabase backend with requests when refreshing the token and when getting the user after oauth redirect